README.md

```diff
@@ -31,11 +31,13 @@ Copy your chosen CSS file to a location on your machine, such as your user folde
 ```
 {
   "vscode_custom_css.imports": [
-    "file:///Users/{your username}/synthwave84.css"
+    "file:///Users/{your username}/.vscode/extensions/robbowen.synthwave-vscode-0.0.7/synthwave84.css"
```

Comment:
DIY and think on _future versions_ of the same package:
`"file:///Users/{your username}/.vscode/extensions/robbowen.synthwave-vscode-{installed version}/synthwave84.css"`